tests/manage/events.spec.js

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from "@playwright/test";
+import AxeBuilder from "@axe-core/playwright";
+import { login } from "../setup/auth";
+
+test("Logged in user can access dashboard", async ({ browser }) => {
```

Comment:
There is a test for that in the [login.spec](https://github.com/EddieHubCommunity/LinkFree/blob/main/tests/login.spec.js).
I don't know if it would be worthwhile to do it on every page as the access logic is the same for all.